### PR TITLE
* Feat Implemented ToC Indicators V2

### PR DESCRIPTION
**Pull request by JuanGuzman-io**

### CODE REVIEW

File: onecgiar-pr-client/src/app/pages/results/pages/result-detail/pages/rd-theory-of-change/components/shared/toc-initiative-out/target-indicator/target-indicator.component.scss [hunk 1]
```diff
@@ -148,44 +102,98 @@ input:disabled {
...
+.bigger-line {
+  background-color: #d9d9d9;
+  width: 100%;
+  height: 10px;
+  border-radius: 0 0 10px 10px;
+  display: flex;
+  align-items: center;
 }
+
+// Borrar
```
Comment (davinzifc) [inline onecgiar-pr-client/src/app/pages/results/pages/result-detail/pages/rd-theory-of-change/components/shared/toc-initiative-out/target-indicator/target-indicator.component.scss:154]: @xKeCo could you check this commented code block

**Review by davinzifc — CHANGES_REQUESTED** [review]
Minor changes such as commented codes and unused functions

File: onecgiar-pr-client/src/app/pages/results/pages/result-detail/pages/rd-theory-of-change/components/shared/toc-initiative-out/target-indicator/target-indicator.component.scss (hunk 1 above)
Comment (xKeCo) [reply onecgiar-pr-client/src/app/pages/results/pages/result-detail/pages/rd-theory-of-change/components/shared/toc-initiative-out/target-indicator/target-indicator.component.scss]: Done ✅

File: onecgiar-pr-server/src/migrations/1701461558478-migrateDataIntoTargetsStringValues.ts [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+    await queryRunner.query(`
+        ALTER TABLE
+            result_indicators_targets
+        DROP
+            COLUMN contributing_indicator_tmp;
+        `);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
```
Comment (davinzifc) [inline onecgiar-pr-server/src/migrations/1701461558478-migrateDataIntoTargetsStringValues.ts:89]: This duplicate code is from a migration which requires retracing your steps to be able to revert the changes.

**Review by davinzifc — APPROVED** [review]
Changes successfully made for the move to production

**Comment by JuanGuzman-io** [comment]
We will do the pull, but keeping in mind @davinzifc comments.
